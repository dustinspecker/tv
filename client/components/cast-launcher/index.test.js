import CastLauncher from '.'
import React from 'react'
import renderer from 'react-test-renderer'

test('CastLauncher shows google-cast-launcher when showLauncher is true', () => {
  const props = {
    showLauncher: true
  }

  const element = <CastLauncher {...props} />

  const component = renderer.create(element)
  expect(component.toJSON()).toMatchSnapshot()
})

test('CastLauncher hides google-cast-launcher when showLauncher is false', () => {
  const props = {
    showLauncher: false
  }

  const element = <CastLauncher {...props} />

  const component = renderer.create(element)
  expect(component.toJSON()).toMatchSnapshot()
})
