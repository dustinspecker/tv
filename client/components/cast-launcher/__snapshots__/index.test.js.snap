// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CastLauncher hides google-cast-launcher when showLauncher is false 1`] = `null`;

exports[`CastLauncher shows google-cast-launcher when showLauncher is true 1`] = `
<div
  id="cast"
>
  <google-cast-launcher />
</div>
`;
